app/models/user.rb

```diff
@@ -1,5 +1,7 @@
 class User < ApplicationRecord
-    has_many :events
-    has_many :invitations 
-    has_many :guests, through: :invitation
+  validates :name, presence: true
+
+  has_many :events
```

Comment:
- [ ] The user should have many events as the `creator`, so you need to add that the `foreign_key` it uses is `creator_id`.